### PR TITLE
Document element selection - grafana/k6#3519

### PR DESCRIPTION
**Pull request by mstoykov**
## What?

Document https://github.com/grafana/k6/pull/3519

## Checklist

Please fill in this template:
- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `make docs` command locally and verified that the changes look good.

Select one of these and delete the others:

If updating the documentation for the most recent release of k6: 
- [ ] I have made my changes in the `docs/sources/next` folder of the documentation.
- [ ] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [ ] I have reflected my changes in the relevant (*e.g.*  when correcting a documentation error) folders of the previous k6 versions of the documentation.

If updating the documentation for the next release of k6:
- [x] I have made my changes in the `docs/sources/next` folder of the documentation.

### CODE REVIEW

File: docs/sources/next/javascript-api/k6-html/element/element-selection.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: 'Element.selectionet()'
```
Comment (joanlopez) [inline docs/sources/next/javascript-api/k6-html/element/element-selection.md:2]:
```suggestion
title: 'Element.selection()'
```

File: docs/sources/next/javascript-api/k6-html/element/_index.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ The Element object provides a similar API to the [DOM Element API](https://devel
 
 | Method                 | Description                                                                                                                                                              |
 | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [selection](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-html/element/element-selection) | returns the selection that is matching the element                                                                                                                       |
```
Comment (olegbespalov) [inline docs/sources/next/javascript-api/k6-html/element/_index.md:19]: Just to align style with another elements in table

```suggestion
| [selection](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-html/element/element-selection) | The selection that is matching the element                                                                                                                       |
```

**Comment by mstoykov** [comment]
I will squash the changes in the second commit after some :+1:

**Review by olegbespalov — APPROVED** [review]

**Review by joanlopez — APPROVED** [review]

File: docs/sources/next/javascript-api/k6-html/element/_index.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ The Element object provides a similar API to the [DOM Element API](https://devel
 
 | Method                 | Description                                                                                                                                                              |
 | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [selection](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-html/element/element-selection) | The selection that is matching the element                                                                                                                       |
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-html/element/_index.md:19]:
```suggestion
| [selection](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-html/element/element-selection) | The selection matching the element.                                                                                                                       |
```

File: docs/sources/next/javascript-api/k6-html/element/element-selection.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+title: 'Element.selection()'
+excerpt: 'Retrieve the Selection matching this element.'
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-html/element/element-selection.md:3]:
```suggestion
description: 'Retrieve the Selection matching this element.'
```

File: docs/sources/next/javascript-api/k6-html/element/element-selection.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Retrieve the Selection matching this element.
+
+Mimics `$(element)`
+
+### Returns
+
+| Type    | Description                          |
+| ------- | ------------------------------------ |
+| Selection | The Selection mathing this element. |
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-html/element/element-selection.md:16]:
```suggestion
| Selection | The Selection matching this element. |
```

**Review by heitortsergent — COMMENTED** [review]
I caught a couple of small typos. Otherwise, it looks good! :shipit:

File: docs/sources/next/javascript-api/k6-html/element/element-selection.md (hunk 4 above)
Comment (mstoykov) [reply docs/sources/next/javascript-api/k6-html/element/element-selection.md:3]: Sorry for the late reply @heitortsergent :bow: 

This is what I just copied from a different page around this docs.

Is this the new name - and we should rename them all, or is it just an alias you prefer? Or something else?

Comment (heitortsergent) [reply docs/sources/next/javascript-api/k6-html/element/element-selection.md:3]: @mstoykov no worries! We should rename them all, it's something I missed during the migration. 😞 

If you can please just change it in this PR, I can open a separate PR to change it in all other files.

**Review by heitortsergent — APPROVED** [review]

**Pull request merged**